content/en/docs/components/central-dash/profiles.md

```diff
@@ -6,13 +6,21 @@ weight = 20
 
 ## What is a Kubeflow Profile?
 
-A Profile is a Kubernetes CRD introduced by Kubeflow that wraps a Kubernetes Namespace.
+A Kubeflow Profile is a [Kubernetes CRD](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) introduced by Kubeflow that wraps a Kubernetes [Namespace](https://kubernetes.io/docs/tasks/administer-cluster/namespaces-walkthrough/). It is designed to solve access management within multi-user kubernetes cluster.
```

Comment:
It's not really correct to say it "solves access management" given the profile does not have any access management aspect, it literally just tells the controller to make a profile.

Access management is sadly done with the strange named RBAC resources, as described in this section.

```suggestion
A Kubeflow Profile is a [Kubernetes CRD](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) introduced by Kubeflow that wraps a Kubernetes [Namespace](https://kubernetes.io/docs/tasks/administer-cluster/namespaces-walkthrough/).
```

Reply:
Also, this should flow into the next paragraph like the current website does, e.g. there should not be a new line here.